### PR TITLE
small language clarifications and deletions

### PR DESCRIPTION
**Pull request by RacheltheEditor**
+ moved around one paragraph out of place in Tax Basics

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -849,12 +848,11 @@ diluted.
...
 growth a startup goes through.
-This is largely reflected in how much funding it has raised—how much ownership, in the
-form of shares, has been sold for capital.
+These stages are largely reflected in how much funding has been raised—how much ownership,
+in the form of shares, has been sold for capital.
 
-The stage of a startup is largely reflected in how much funding it has raised.
 Very roughly,
-[typical levels](http://blog.eladgil.com/2011/03/how-funding-rounds-differ-seed-series.html)
+[typical stages](http://blog.eladgil.com/2011/03/how-funding-rounds-differ-seed-series.html)
```
Comment (jlevy) [inline README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -1251,25 +1248,28 @@ But this window is not always open.
...
-    options (as illustrated [many companies](https://github.com/holman/extended-exercise-windows)
-    companies now choosing to do).
-  - On balance, it seems many people 🔑[didn’t understand](https://medium.com/@ben_mathes/90-days-and-my-six-figure-mistake-a495f4a188e2)
-    the nuances of this before joining companies in the past, but with the risks of short
-    exercise windows for employees becoming more widely known, the recent trend has shifted to
-    longer exercise windows.
+  - As a side note, some also argue that it’s a legal requirement of ISOs to have a 90-day
+    exercise window.
+    While this is technically true, it’s
+    [not the whole story](https://news.ycombinator.com/item?id=9254299). It is possible for
```
Comment (jlevy) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -1700,11 +1699,11 @@ regarding stock awards and stock options.
   [Section 83(b)](https://www.law.cornell.edu/uscode/text/26/83), offers an alternative, called
   an
   [**83(b) election**](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/),
-  that will protect you from high potential tax at time of vesting, which ensures that
-  you’ll be taxed on the receipt of the “property” (the stock), rather than at the time the
-  stock vests. With an 83(b) election, you’re telling the IRS you want to pay taxes early,
-  and this can potentially reduce your tax significantly:
-  If the shares go up in value, the taxes owed on vesting might be far greater than the
+  that will protect you from high potential tax at time of vesting, ensuring you’ll be taxed
+  on the receipt of the “property” (the stock), rather than at the time the stock vests.
```
Comment (jlevy) [inline README.md]: Perhaps remove quotes?

File: README.md [hunk 4]
```diff
@@ -1721,7 +1720,7 @@ regarding stock awards and stock options.
   Technically, it cannot be made on the receipt of a stock *option* itself:
   You first must exercise that option, then file the election.
 - If you receive an early exercisable stock option (when you don’t have to wait for the the
-  stock to vest), you can make an 83(b) election on receipt of the shares on exercise.
+  stock to vest), you can make an 83(b) election at receipt of the exercised shares.
```
Comment (jlevy) [inline README.md]: I believe every lawyer would say "on" or "upon" here, not "at".

File: README.md [hunk 5]
```diff
@@ -1848,8 +1847,8 @@ The AMT trap does not apply to NSOs.
 
 ### Stock awards vs ISOs vs NSOs
 
-Because the differences are so nuanced, what follows is a summary of the differences
-between restricted stock awards, ISOs and NSOs from an employee’s point of view.
+Because the differences are so nuanced, what follows is a summary of the taxes on
+restricted stock awards, ISOs, and NSOs, from an employee’s point of view.
```
Comment (jlevy) [inline README.md]: Oxford forever!

File: README.md [hunk 6]
```diff
@@ -1414,10 +1405,18 @@ meanings we’ll do our best to spell out.
 
 ### Kinds of income
 
-🄳 You must pay federal, state, and in some cases, local **income taxes** on your **income**,
-which is the money you make.
+🄳 You must pay federal, state, and in some cases, local **income taxes** on the money you
```
Comment (jlevy) [inline README.md]: will have to resolve conflicts here

File: README.md [hunk 7]
```diff
@@ -2256,8 +2255,8 @@ A Guide like this can’t give you personalized advice on what a reasonable offe
 that depends greatly on your skills, the marketplace of candidates, what other offers you
 have, what the company can pay, what other candidates the company has found, and the
 company’s needs.
-However, we will cover the basics of what to expect with offers, and how to approach
-negotiations.
+But we can cover the basics of what to expect with offers, and provide some expert advice
```
Comment (jlevy) [inline README.md]: Nit: is "expert" necessary? I usually get suspicious if someone tells me they're giving expert advice.

File: README.md (hunk 3 above)
Comment (jlevy) [reply README.md]: No strong opinion tho.

File: README.md [hunk 8]
```diff
@@ -564,9 +564,8 @@ paid dividends.
 If you’re considering working for a startup, what we cover next on how these early-stage
 companies raise money and grow is helpful in understanding what your equity may be worth.
 
-[How equity is granted](#how-equity-is-granted) differs between startups and established
-companies. If you are only concerned with large and established companies, you can
-[skip ahead](#equity-compensation-basics).
+If you’re only concerned with large and established companies, you can skip ahead to
+[how equity is granted](#ehow-equity-is-granted).
```
[the remark this answers is not in the document]
Comment (RacheltheEditor) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (RacheltheEditor) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (RacheltheEditor) [reply README.md]: Fixed ("upon")

File: README.md (hunk 5 above)
Comment (RacheltheEditor) [reply README.md]: 😂

**Pull request merged**